docs/dptp-triage-sop/openshift-priv.md

```diff
@@ -0,0 +1,45 @@
+# `openshift-priv` Organization
+
+## Unrelated histories
+
+A force-push in one of the repositories which has a counterpart in
+`openshfit-priv` will cause the job to fail to synchronize the repositories,
+e.g.:
+
+```
+time="2022-11-01T08:23:40Z" level=warning msg="error occurred while fetching remote and merge" branch=release-4.13 destination=openshift-priv/cluster-api-provider-ibmcloud@release-4.13 error="[failed to merge openshift-cluster-api-provider-ibmcloud/release-4.13: failed with 128 exit-code: fatal: refusing to merge unrelated histories\n, failed to perform merge --abort: failed with 128 exit-code: fatal: There is no merge to abort (MERGE_HEAD missing).\n]" local-repo=/tmp/1970879496/openshift/cluster-api-provider-ibmcloud org=openshift repo=cluster-api-provider-ibmcloud source=openshift/cluster-api-provider-ibmcloud@release-4.13 source-file=openshift-cluster-api-provider-ibmcloud-release-4.13.yaml variant=
+```
+https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-openshift-release-private-org-sync/1587355868737835008
+
+Because merge conflicts are [ignored][private_org_sync_readme] by
+`private-org-sync`, this will not cause the synchronization job to fail;
+instead, the two repositories will silently diverge.  Eventually, this will
```

Comment:
> the two repositories will silently diverge

I fail to understand the why of this choice; it's not even explained here [DPTP-1426](https://issues.redhat.com/browse/DPTP-1426)

Reply:
> The reason being that a merge conflict is a standard scenario that can arise in the CVE workflow and does not require administrative action